home-screen-widget/profiles/README.md

```diff
@@ -0,0 +1,23 @@
+#Home Screen Example Widget
+
+This is a guided example on how to develop a home screen widget.
+In this example we provide a working widget that uses the Beekeeper Javascript SDK to fetch 50 user profiles and display
+them in a grid layout.
```

Comment:
Can you include a photo of the final state here? Maybe add a ``/photos`` folder and add screenshots there.